src/components/note/NoteModal.tsx

```
@@ -1,53 +1,57 @@
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faFlag, faXmark } from '@fortawesome/free-solid-svg-icons';
+import { faFlag, faLink, faXmark } from '@fortawesome/free-solid-svg-icons';
 import { motion, AnimatePresence } from 'framer-motion';
-import ReactDOM from 'react-dom';
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+import Link from 'next/link';
+import { createPortal } from 'react-dom';
 import { useClickOutside } from '@/hooks/useClickOutside';
 import { useGoals } from '@/hooks/useGoals';
```

ESLint

[error] 'useGoals' is defined but never used.

(no-unused-vars)